src/utils.rs

```diff
@@ -484,6 +485,14 @@ pub fn isatty() -> bool {
     }
 }
 
+pub fn iscolored(color: Color) -> bool {
```

Comment:
I think `use_colored_tty` would be a better name.

Reply:
agreed, thanks.